### DISCUSSION TITLE
Renovate app disabled on gitlab.com

### DISCUSSION THREAD

**Discussion by kevincox** [renovatebot/renovate#12798] [Request Help]
### What would you like Renovate to be able to do?

Use hosted Renovate Bot on gitlab.com

Was:
- https://github.com/renovatebot/renovate/issues/7843
- https://github.com/renovatebot/app-support/issues/106

GitLab now believes that this is unblocked (https://gitlab.com/gitlab-org/gitlab/-/issues/11561#note_740498950) due to https://gitlab.com/gitlab-org/gitlab/-/issues/328553.

It would be interesting to reevaluate the security here and re-enable the bot if possible.

### Is this a feature you are interested in implementing yourself?

No

**Comment by viceice** [comment]
Still blocked, see https://gitlab.com/gitlab-org/gitlab/-/issues/346298

**Reply by meshantz** [reply]
Looks like that issue was closed last week. Does that mean work is underway (or planned to be underway soon) to restore this?

https://gitlab.com/gitlab-org/gitlab/-/issues/346298#note_2dd272ba0712c32a8b29689bb16b294222252b1d

**Reply by viceice** [reply]
yes, well probably enable the app when all work is done. currently the UI is missing, so you need some graphql foo to configure it right. we'll wait a bit longer until it's fully functional, but maybe @rarkins will have a look in next weeks.
